api/db/repository/petition.py

```diff
@@ -62,12 +76,90 @@ def get_by_id(_db: Session, petition_id: int, petition_status=""):
     return db_petition.first()
 
 
-def get_all(_db: Session, offeset: int = 0, limit: int = 100):
-    return _db.query(Petition).offset(offeset).limit(limit).all()
+def get_all(
+    _db: Session,
+    offeset: int = 0,
+    limit: int = 100,
+    search_query: str = "",
+    filtering: FilteringRequest = None,
+    ordering: List[OrderingRequest] = None,
+):
+    query = _db.query(Petition).filter(
+        Petition.title.contains(search_query)
```

Comment:
Same here with wildcards

Reply:
I fixed it



.................................